sdk/containerregistry/azure-containerregistry/setup.py

```diff
@@ -58,6 +59,9 @@
             "azure",
         ]
     ),
+    package_data={
```

Comment:
Seems like we don't need this.

MANIFEST.in + include_package_data=True, is good enough.